Source/Anfangko.cs

```diff
@@ -322,50 +322,61 @@ public static void Calculate()
                                 }
 
                                 double lang = Math.Sqrt(Math.Pow(Program.LS_X2[i] - Program.LS_X1[i], 2) + Math.Pow(Program.LS_Y2[i] - Program.LS_Y1[i], 2));
-
-                                zuff1 = Rng.NextDouble();
-
-                                double zahl0 = lang * zuff1;
-                                double zahl1 = 0;
-                                //noise abatement wall +1m
-                                if (Program.LS_Laerm[i] > 0)
+                                double zzz = 0;   // z value
+                                double zahl1 = 0; // vertical mixing height
+                                
+                                if (lang > 0.1) 
```

Comment:
default case: the line source has a lenght > 0 -> calculate the height values along the rising or descending line